docs/MANUAL.md

```diff
@@ -1027,6 +1027,14 @@ ProxyPass /webirc http://127.0.0.1:8067 upgrade=websocket
 ProxyPassReverse /webirc http://127.0.0.1:8067
 ```
 
+On Caddy, websocket proxying can be configured with:
+
+```
+handle_path /webirc {
+  reverse_proxy ircd:8067
```

Comment:
`ircd` is the hostname? `127.0.0.1:8067` would be consistent with the other examples

Reply:
Good catch, thank you. I've updated with the consistent IP address.